util/gce/gcs.cc

```diff
@@ -318,10 +318,13 @@ template <typename RespBody> Status GCS::SendWithToken(Request* req, Response<Re
 
     if (IsUnauthorized(*resp)) {
       RETURN_IF_ERROR(RefreshToken(req));
-      *resp = Response<RespBody>{};
-      continue;
     }
+    *resp = Response<RespBody>{};
```

Comment:
Here is my change.. When calling List we once in a while get 503 error. I wanted to check if retries will help. before the fatal log was inside the for loop and we crash after the first try